OpenRA.Mods.Common/Lint/CheckSequences.cs

```diff
@@ -22,12 +22,14 @@ public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
 			var sequences = MiniYaml.MergeLiberal(map.SequenceDefinitions, Game.ModData.Manifest.Sequences.Select(s => MiniYaml.FromFile(s)).Aggregate(MiniYaml.MergeLiberal));
 
 			foreach (var actorInfo in map.Rules.Actors)
-				foreach (var renderInfo in actorInfo.Value.Traits.WithInterface<RenderSimpleInfo>())
+			{
+				foreach (var renderInfo in actorInfo.Value.Traits.WithInterface<RenderSpritesInfo>())
 				{
 					var image = renderInfo.Image ?? actorInfo.Value.Name;
```

Comment:
I believe this is what @pchote meant by
>This needs to use RenderSpritesInfo and query the image name from GetImage(). The render traits moved on from renderInfo.Image ?? actorInfo.Value.Name ages ago.

?
Although I don't see why `RenderSimpleInfo` needs to change to `RenderSpritesInfo` to use `.GetImage()`.